source/array-buffer.js

```diff
@@ -1,11 +1,11 @@
 import {getStreamContents} from './contents.js';
-import {throwObjectStream, getLengthProp} from './utils.js';
+import {noop, throwObjectStream, getLengthProp} from './utils.js';
 
 export async function getStreamAsArrayBuffer(stream, options) {
 	return getStreamContents(stream, arrayBufferMethods, options);
 }
 
-const initArrayBuffer = () => new Uint8Array(0);
+const initArrayBuffer = () => ({contents: new Uint8Array(0)});
```

Comment:
The changes in this file and the following ones are related to refactoring stateful variables into a `state` object. See comment below.